ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -1818,6 +1818,41 @@ def test_paddle_any(
     )
 
 
+# floor_divide
+@handle_frontend_test(
+    fn_tree="paddle.tensor.math.floor_divide",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Similarly, does this function only works on `"float"`?

Reply:
I have solved this issue.

Reply:
I updated the changes and pushed the files. Check it